Microsoft.Azure.Cosmos/src/EncodingExtensions.cs

```diff
@@ -38,7 +38,12 @@ public static unsafe int GetChars(this Encoding encoding, ReadOnlySpan<byte> src
             }
         }
 
-        public static unsafe int GetBytes(this Encoding encoding, string src, Span<byte> dest)
+        public static int GetBytes(this Encoding encoding, string src, Span<byte> dest)
+        {
+            return encoding.GetBytes(src, src.Length, dest, dest.Length);
+        }
+
+        public static unsafe int GetBytes(this Encoding encoding, string src, int charCount, Span<byte> dest, int byteCount)
```

Comment:
```suggestion
        public static unsafe int GetBytes(this Encoding encoding, ReadOnlySpan<char> src, Span<byte> dest)
```
Same overload .net Standard 2.1 and .Net Core 3.* are adding

Reply:
Nice!